src/current/cockroachcloud/production-checklist.md

```diff
@@ -70,27 +77,22 @@ Production clusters should not authorize `0.0.0.0/0`, which allows all networks.
 
 For enhanced network security and reduced network latency, you can set up private connectivity so that inbound connections to your cluster from your cloud tenant are made over the cloud provider's private network rather than over the public internet. For CockroachDB {{ site.data.products.dedicated }} clusters deployed on GCP, refer to [Google Cloud Platform (GCP) Virtual Private Cloud (VPC) peering]({% link cockroachcloud/network-authorization.md %}#vpc-peering). For CockroachDB {{ site.data.products.dedicated }} clusters or multi-region CockroachDB {{ site.data.products.serverless }} clusters deployed on AWS, refer to [Amazon Web Service (AWS) PrivateLink]({% link cockroachcloud/network-authorization.md %}#aws-privatelink).
 
-### Use a pool of persistent connections
-
-Creating the appropriate size pool of connections is critical to gaining maximum performance in an application. Too few connections in the pool will result in high latency as each operation waits for a connection to open up. But adding too many connections to the pool can also result in high latency as each connection thread is being run in parallel by the system. The time it takes for many threads to complete in parallel is typically higher than the time it takes a smaller number of threads to run sequentially.
-
-For guidance on sizing, validating, and using connection pools with CockroachDB, refer to [Use Connection Pools](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/connection-pooling).
-
 <a id="keeping-connections-current"></a>
 ### Keep connections current
```

Comment:
Let's merge Keeping Connections Current and SQL Connection Handling.

Reply:
Done with some wording adjustments.

Reply:
I'd restructure this a little bit. The top level title could be "SQL connection Handling" with a subsection on "connection pooling" since this is performance related, then a section for "keeping connections current" since this is for reliability.

Reply:
Apologies, I misinterpreted what you meant by "merge". Fixed.